### ISSUE TITLE
Need absolute differences when comparing content_frame_time_reason data points

### ISSUE THREAD

**Issue by mstange** [mozilla/glam#1136]
When comparing [two datapoints for the `content_frame_time_reason` probe](https://glam.telemetry.mozilla.org/firefox/probe/content_frame_time_reason/explore?activeBuckets=%5B%22OnTime%22%2C%22SlowComposite%22%2C%22MissedCompositeLong%22%2C%22MissedCompositeMid%22%2C%22MissedComposite%22%2C%22MissedCompositeLow%22%2C%22NoVsyncNoId%22%2C%22NoVsync%22%5D&process=parent&ref=20210115035053), the "DIFF" column in the summary table is showing useless values.

<img width="342" alt="Screen Shot 2021-01-26 at 12 37 40 PM" src="https://user-images.githubusercontent.com/961291/105882339-88099680-5fd3-11eb-9b4a-29a5d0583472.png">

Basically, we're comparing two pie charts here. The "OnTime" percentage has gone down from 82.26% to 78.76%, and I'm trying to find out where those 3.5% percentage points went.

So those 3.5% must somehow be absorbed by the other rows.
A table with absolute differences would be much more useful:

| Category | Hov | Ref | Diff |
|---|---|---|---|
| OnTime | 82.26% | 78.76% | -3.5% |
| SlowComposite | 5.36% | 3.57% | +1.79% |
| MissedCompositeLong | ... | ... | ... |
| ... | ... | ... | ... |

(The sum of the Diff column should be 0%, because each column individually sums up to 100%.)
